packages/loot-core/src/server/accounts/title/lower-case.js

```diff
@@ -1,18 +1,18 @@
 const conjunctions = [
-  'for',
+  'for', //
```

Comment:
```suggestion
  'for',
```

Reply:
This is intentional to keep each element on a new line. https://github.com/prettier/prettier-vscode/issues/352#issuecomment-428714849